packages/server/modules/multiregion/tests/serverAdmin.graph.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { MultiRegionConfig } from '@/modules/multiregion/domain/types'
+import { Regions } from '@/modules/multiregion/repositories'
+import { BasicTestUser, createTestUser } from '@/test/authHelper'
+import {
+  CreateNewRegionDocument,
+  CreateServerRegionInput,
+  GetAvailableRegionKeysDocument,
+  GetRegionsDocument
+} from '@/test/graphql/generated/graphql'
+import { testApolloServer, TestApolloServer } from '@/test/graphqlHelper'
+import { beforeEachContext, truncateTables } from '@/test/hooks'
+import { MultiRegionConfigServiceMock } from '@/test/mocks/global'
+import { Roles } from '@speckle/shared'
+import { expect } from 'chai'
+
+describe('Multi Region Server Settings', () => {
+  let testAdminUser: BasicTestUser
+  let apollo: TestApolloServer
+
+  const fakeRegionKey1 = 'us-west-1'
+  const fakeRegionKey2 = 'eu-east-2'
+
+  const fakeRegionConfig: MultiRegionConfig = {
+    [fakeRegionKey1]: {
+      postgres: {
+        connectionUri: 'postgres://user:password@uswest1:port/dbname'
+      }
+    },
+    [fakeRegionKey2]: {
+      postgres: {
+        connectionUri: 'postgres://user:password@eueast3:port/dbname'
+      }
+    }
+  }
+
+  before(async () => {
+    // Have to mock both
+    MultiRegionConfigServiceMock.mockFunction(
```

Comment:
I could've gone the DI route, but that would've been a unit test and I wanted a full integration test. IMO this is the perfect use case for mocks, and I don't really see a cleaner way to do this